contracts/JBProjects.sol

```diff
@@ -140,18 +133,17 @@ contract JBProjects is ERC721Votes, Ownable, IJBProjects, JBOperatable {
   */
   function tokenURI(uint256 _projectId) public view override returns (string memory) {
```

Comment:
^ here we use all caps (my preference)

Reply:
this is overriden and part of the base contract. everywhere else we use Uri